### PR TITLE
trigger drone publish on tag event

### PR DESCRIPTION
**Pull request by meldafrawi**
Signed-off-by: Mohamed Eldafrawi <mohamed.eldafrawi@rancher.com>

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -34,9 +40,27 @@ steps:
     event:
     - push
 
+- name: build-with-tag
+  pull: default
+  image: rancher/dapper:1.11.2
+  commands:
+  - VERSION="${DRONE_TAG}" dapper ci
```
Comment (yasker) [inline .drone.yml:47]: I think `build` should pick up the version from git repo automatically?

Comment (meldafrawi) [reply .drone.yml:47]: It will pickup git commit not the tag.

Comment (yasker) [reply .drone.yml:47]: that's weird. e.g. if you try it locally, it should pick up the tag. see `scripts/version`

Comment (meldafrawi) [reply .drone.yml]: fixed

**Pull request merged**